uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -506,15 +530,16 @@ define([
      */
     function setCalendarProps(contactID, calendar) {
       var leaveRequest,
-        indexedByMonth = vm.months.map(function () {
-          return [];
+        monthObject,
```

Comment:
You don't need this defined here, you are using only in the in the `_.each` callback